bindata/manifests/metrics-exporter/metrics-prometheus-rule.yaml

```diff
@@ -0,0 +1,38 @@
+---
+{{ if .IsPrometheusOperatorInstalled }}
```

Comment:
i recall we discussed having a separate env var for deploying rules right ?

from aug 06 2024 meeting notes:
> Adding a new env variable to decide if deploy prometheus rules

Reply:
yes, I'm fixing that